_pages/04_schedule.md

```diff
@@ -26,20 +26,27 @@ The following is a final schedule for the workshop.
 | Closing remarks                    | 17:40-17:45 |
 
 
+## Meta-analyses
+
+All Meta-analyses are for 10 minutes.
+
+- Smarter prototyping for neural learning (Prabhu Pradhan)
+- Advances in deep learning for skin cancer detection (Andre Pacheco)
+- Unsupervised minimax (Juergen Schmidhuber)
+
 ## Lightning Talks
```

Comment:
I think we should call these "Retrospective lightning talks"

Reply:
done